tests/_conftest.py

```diff
@@ -6,8 +6,7 @@
 import sqlalchemy as sa
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-from pytest_postgresql.factories import (init_postgresql_database,
-                                         drop_postgresql_database)
+from pytest_postgresql.factories import DatabaseJanitor
```

Comment:
The import is wrong. This is correct:

    from pytest_postgresql.janitor import DatabaseJanitor